package.json

```diff
@@ -6,33 +6,33 @@
   "author": "Mate Academy",
   "license": "GPL-3.0",
   "dependencies": {
-    "bulma": "^0.9.3",
+    "bulma": "^0.9.4",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-scripts": "^4.0.3"
   },
   "devDependencies": {
-    "@cypress/react": "^5.12.4",
+    "@cypress/react": "^5.12.5",
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.